Sources/Commands/SymbolGraphExtract.swift

```diff
@@ -65,6 +79,16 @@ public struct SymbolGraphExtract {
     }
 }
 
+/// Access control levels.
+public enum AccessLevel: String, RawRepresentable, CustomStringConvertible, CaseIterable {
+	// The cases reflect those found in `include/swift/AST/AttrKind.h` of the swift compiler (at commit 03f55d7bb4204ca54841218eb7cc175ae798e3bd)
```

Comment:
What is the failure mode if the compiler introduces a new one?  Is it just that it won't be expressible in this tool until it's been updated, or will something break?

Reply:
It won't be expressible in this tool until it's been updated.

The only place where I use that access level is for the argument `--minimum-access-level`. When you would pass it an access level that it does not now yet, this tool (`swift package dump-symbol-graph`) will throw an error like the following one:
```sh
error: The value 'futurelevel' is invalid for '--minimum-access-level <minimum-access-level>'
Help:  --minimum-access-level <minimum-access-level>  Include symbols with this access level or more. Possible values: private | fileprivate | internal | public | open
Usage: swift package dump-symbol-graph <options>
  See 'package dump-symbol-graph -help' for more information.
```

Reply:
Sounds great — the main thing I was concerned about was whether this code could fail if the enum changes, but it sounds as if that's not possible.  Thanks!